### ISSUE TITLE
Play button prioritizes FPS, should offer option to play each frame

### ISSUE THREAD

**Issue by aphilip442** [napari/napari#2559]
## 🐛 Bug

When a large stack is loaded lazily, the play button loads frames in seemingly random order in viewer.

Example sequence: (start at 30) 30-31-32-42-72

## To Reproduce

1. Load Files as Stack
2. Hit Play.
3. Frames are not loaded sequentially

## Expected behavior

Frames should be loaded sequentially.

## Environment

napari: 0.4.7
Platform: Linux-5.8.0-49-generic-x86_64-with-glibc2.31
System: Ubuntu 20.04.2 LTS
Python: 3.9.2 | packaged by conda-forge | (default, Feb 21 2021, 05:02:46) [GCC 9.3.0]
Qt: 5.12.9
PyQt5: 5.12.3
NumPy: 1.20.2
SciPy: 1.6.2
Dask: 2021.04.0
VisPy: 0.6.6

OpenGL:

GL version: 3.1 Mesa 20.2.6
MAX_TEXTURE_SIZE: 16384
Screens:

screen 1: resolution 1280x800, scale 1.0
Plugins:

console: 0.0.3
svg: 0.1.4

## Additional context

I installed a VM Ubuntu on Windows server to do further testing following https://github.com/napari/napari/issues/2489.

**Comment by tlambert03** [comment]
The play button is designed to maintain the requested FPS (not to block until lazily loaded data is ready).  I can see that it would also be nice to have a mode that waits for lazy data no matter what, but one thing you can do in the meantime is to decrease the frame rate (right click on the play button and decrease FPS).

**Comment by aphilip442** [comment]
@tlambert03 okay, that makes more sense now.
Maybe this should be added in the tooltip?
Alternatively, the play button could have 2 behaviours (adjustable in preferences), as maintaining frame rate is not something that can be reasonably achieved with lazy loading.

**Comment by tlambert03** [comment]
Agreed: “prioritize FPS (may skip frames if necessary)” and “show all frames (playback rate will depend on data)”

**Comment by sofroniewn** [comment]
> Agreed: “prioritize FPS (may skip frames if necessary)” and “show all frames (playback rate will depend on data)”

Seems like a good option for our settings. I'm supportive

**Comment by psobolewskiPhD** [comment]
I updated the title and the label to clarify that current play button behavior is expected, but a feature of playing every frame, even is slow, would be nice!
